packages/@aws-cdk/aws-apigatewayv2/test/http/api.test.ts

```diff
@@ -531,6 +532,63 @@ describe('HttpApi', () => {
   });
 });
 
+describe('SpecHttpApi', () => {
+  test('default', () => {
```

Comment:
I'd like to see more unit test cases overall. Looks like there aren't any for `api-definition`.